app/src/main/java/com/everest/moviedb/model/MovieService.kt

```diff
@@ -0,0 +1,18 @@
+package com.everest.moviedb.model
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface MovieService {
+
+    @GET("movie/popular?api_key=4fcef1da977950737a961196b2c901e0&language=en-US&page=1")
```

Comment:
we should not hardcode API key like this.
check `Interceptors` for OkHTTP. we can dynamically add API key to request.